js/client-chat.js

```diff
@@ -1683,8 +1654,8 @@
 		},
 		comparator: function (a, b) {
 			if (a === b) return 0;
-			var aRank = (this.rankOrder[this.room.users[a] ? this.room.users[a].substr(0, 1) : ' '] || 6);
-			var bRank = (this.rankOrder[this.room.users[b] ? this.room.users[b].substr(0, 1) : ' '] || 6);
+			var aRank = (Config.groups[this.room.users[a] ? this.room.users[a].substr(0, 1) : Config.defaultGroup || ' '] || {order: (Config.defaultOrder || 6)}).order;
```

Comment:
`.substr(0, 1)` should be `.charAt(0)`

(This code was originally from when I kept on forgetting `.charAt` existed)